packages/esm-patient-medications-app/src/order-basket/order-basket.component.tsx

```diff
@@ -46,6 +53,29 @@ const OrderBasket = connect<OrderBasketProps, OrderBasketStoreActions, OrderBask
     isValidating,
   } = usePatientOrders(patientUuid, 'ACTIVE', config.careSettingUuid);
 
+  const encounterCreation = () => {
+    const abortController = new AbortController();
+    createEmptyEncounter(patientUuid, sessionObject, config, abortController).then(
+      ({ data }) => {
+        if (data) {
+          setEncounterUuid(data?.uuid);
+          setIsLoading(false);
+        }
+      },
+      (error) => {
+        const apiError = error.responseBody.error.message;
+        const noProviderError = "[could not execute statement => Column 'provider_id' cannot be null]";
+        setIsLoading(false);
+        showNotification({
+          title: t('noProviderError', 'Error creating an encounter'),
+          kind: 'error',
```

Comment:
I was wondering whether this would look better as a warning just.

Reply:
It would look like this.
<img width="1127" alt="Screenshot 2022-11-30 at 02 02 58" src="https://user-images.githubusercontent.com/30952856/204668532-2734eaa1-820c-4906-bf12-4d3eee2c8672.png">

Reply:
No, it should be an error.